content/sensu-go/5.11/installation/install-sensu.md

```diff
@@ -5,385 +5,330 @@ description: "Sensu Go is available for Linux, Windows (agent and CLI only), mac
+# Pull the image based on Red Hat Enterprise Linux
+docker pull sensu/sensu-rhel
 {{< /highlight >}}
 
-Install the `sensu-go-agent` package.
+{{< highlight "Ubuntu/Debian" >}}
+# Add the Sensu repository
+curl -s https://packagecloud.io/install/repositories/sensu/stable/script.deb.sh | sudo bash
 
-{{< highlight shell >}}
-sudo apt-get install sensu-go-agent
+# Install the sensu-go-backend package
+sudo apt-get install sensu-go-backend
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
-
-{{< platformBlock "RHEL/CentOS" >}}
-#### RHEL/CentOS
-
-Add the Sensu repository.
-
-{{< highlight shell >}}
+{{< highlight "RHEL/CentOS" >}}
+# Add the Sensu repository
 curl -s https://packagecloud.io/install/repositories/sensu/stable/script.rpm.sh | sudo bash
-{{< /highlight >}}
-
-Install the `sensu-go-agent` package.
 
-{{< highlight shell >}}
-sudo yum install sensu-go-agent
+# Install the sensu-go-backend package
+sudo yum install sensu-go-backend
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
-
-{{< platformBlock "Windows" >}}
+{{< /language-toggle >}}
 
-#### Windows {#windows-agent}
+##### 2. Configure and start
 
-Download the Sensu agent for Windows [`amd64`](https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1.5015_en-US.x64.msi) or [`386`](https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1_5015_en-US.x86.msi) architectures.
+You can configure Sensu using `sensu-backend start` flags or an /etc/sensu/backend.yml file, the former taking precedence.
+At a minimum, the Sensu backend requires the `state-dir` flag, but here are some other useful configs and templates.
 
-{{< highlight text >}}
-Invoke-WebRequest https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.11.1/sensu-go-agent_5.11.1.5015_en-US.x64.msi  -OutFile "$env:userprofile\sensu-go-agent_5.11.1.5015_en-US.x64.msi"
-{{< /highlight >}}
+{{< language-toggle >}}
 
-Start the installation wizard.
-
-{{< highlight text >}}
-msiexec.exe /i $env:userprofile\sensu-go-agent_5.11.1.5015_en-US.x64.msi
+{{< highlight Docker >}}
+docker run -v /var/lib/sensu:/var/lib/sensu -d --name sensu-backend -p 2380:2380 -p 3000:3000 -p 8080:8080 -p 8081:8081 sensu/sensu:latest sensu-backend start
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
-
-{{< platformBlock "Ubuntu/Debian RHEL/CentOS Windows" >}}
-
-### 2. Create the configuration file
-
-{{< platformBlockClose >}}
-
-{{< platformBlock "Ubuntu/Debian RHEL/CentOS" >}}
-
-#### Linux
-
-Copy the example agent config file to the default config path.
+{{< highlight "Docker Compose" >}}
+---
+version: "3"
+services:
+  sensu-backend:
+    image: sensu/sensu:latest
+    ports:
+    - 3000:3000
+    - 8080:8080
+    - 8081:8081
+    volumes:
+    - "sensu-backend-data:/var/lib/sensu/etcd"
+    command: "sensu-backend start --state-dir /var/lib/sensu/sensu-backend --log-level debug"
+
+volumes:
+  sensu-backend-data:
+    driver: local
+{{< /highlight >}}
```

Comment:
To do:

- [ ] Make sure `docker run` example is correct and reflects best practices
- [ ] Make sure Docker Compose example is consistent with `docker run` example